hack/parallel.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+NAMESPACE=${KUBEVIRT_INSTALLED_NAMESPACE:-kubevirt}
+
+kubectl() { cluster-up/kubectl.sh "$@"; }
+
+kubectl get nodes >/dev/null 2>&1
+if [ $? -ne 0 ]; then
+    echo "Cluster not found, exiting"
+    exit 1
+fi
+
+kubectl get ns $NAMESPACE >/dev/null 2>&1
```

Comment:
This code probably should go into `hack/cluster-clean.sh` directly and add a NOOP
where needed because there might be other operations that will be added to `cluster-clean` and they will not be aware of this code here.

Reply:
its forbidden to add it there, because cluster-clean should perform all the cleaning, 
sometimes leftovers can exists, 
this one acts as a preliminary condition, for local use, where you control your cluster